### PR TITLE
feat: style individual menu bar items

### PR DESCRIPTION
**Pull request by yuriy-fix**
In progress. Please don't merge!

### CODE REVIEW

File: articles/ds/components/menu-bar/index.asciidoc [hunk 1]
```diff
@@ -67,6 +69,39 @@ Note that the standard Menu Button styles can be adjusted using theme features,
 // TODO: restore after porting foundation docs, replace line above
 // Note that the standard Menu Button styles can be adjusted using <<../../foundation/color#,theme features>>, so these variants should only be used to differentiate special instances of the component.
 
+=== Individual Menu Items Themes
```
Comment (jouni) [inline articles/ds/components/menu-bar/index.asciidoc:72]:
```suggestion
=== Styling Menu Items
```

File: articles/ds/components/menu-bar/index.asciidoc [hunk 2]
```diff
@@ -67,6 +69,39 @@ Note that the standard Menu Button styles can be adjusted using theme features,
...
+
+Individual menu item's appearance can be adjusted by applying <<../../customization/custom-theme#, custom theme>> to it.
```
Comment (jouni) [inline articles/ds/components/menu-bar/index.asciidoc:74]:
```suggestion
An individual menu item can be style using a <<../../customization/component-variants#custom-variants, custom variant>>.
```

File: articles/ds/components/menu-bar/index.asciidoc [hunk 3]
```diff
@@ -67,6 +69,39 @@ Note that the standard Menu Button styles can be adjusted using theme features,
...
+
+To style root level item theme should be created for `vaadin-menu-bar-button`.
```
Comment (jouni) [inline articles/ds/components/menu-bar/index.asciidoc:76]:
```suggestion
To style a root level item, create a custom variant for the `vaadin-menu-bar-button` component.
```

File: articles/ds/components/menu-bar/index.asciidoc [hunk 4]
```diff
@@ -67,6 +69,39 @@ Note that the standard Menu Button styles can be adjusted using theme features,
...
+
+To style sub-menu item theme should be created for `vaadin-context-menu-item`.
```
Comment (jouni) [inline articles/ds/components/menu-bar/index.asciidoc:78]:
```suggestion
To style a sub-menu item, created a custom variant for the `vaadin-context-menu-item` component.

See <<../../customization/custom-theme#vaadin-component-styles, Vaadin Component Variants>> for details how to add style sheets for those components.
```

Comment (dougjewsbury) [reply articles/ds/components/menu-bar/index.asciidoc:78]: Just a typo: "created" -> "create".

File: articles/ds/components/menu-bar/index.asciidoc (hunk 2 above)
Comment (dougjewsbury) [reply articles/ds/components/menu-bar/index.asciidoc:74]: Just a typo: "style" -> "styled".

File: articles/ds/components/menu-bar/index.asciidoc (hunk 3 above)
Comment (dougjewsbury) [reply articles/ds/components/menu-bar/index.asciidoc:76]: Suggest "root level" -> "root-level".

**Review by dougjewsbury — COMMENTED** [review]
AsciiDoc parts look good to me.

File: articles/ds/components/menu-bar/index.asciidoc (hunk 2 above)
Comment (jouni) [reply articles/ds/components/menu-bar/index.asciidoc:74]: Fixed

File: articles/ds/components/menu-bar/index.asciidoc (hunk 3 above)
Comment (jouni) [reply articles/ds/components/menu-bar/index.asciidoc:76]: Updated

File: articles/ds/components/menu-bar/index.asciidoc (hunk 4 above)
Comment (jouni) [reply articles/ds/components/menu-bar/index.asciidoc:78]: Fixed

**Review by jouni — APPROVED** [review]

**Comment by jouni** [comment]
@yuriy-fix, is this PR going anywhere?

**Comment by jouni** [comment]
Please sign the CLA.

**Review by jouni — APPROVED** [review]

**Pull request merged**